bundles/mapping/mapmodule/MapModuleTextButton.jsx

```diff
@@ -81,16 +81,18 @@ export const MapModuleTextButton = ({ visible, onClick, icon, children, active,
         return null;
     };
 
-    return <ThemedButton
-        icon={icon || null}
-        onClick={onClick}
-        active={active}
-        position={position}
-        loading={loading}
-        {...rest}
-    >
-        {children}
-    </ThemedButton>;
+    return <div>
```

Comment:
Add a comment here that the div is so publisher can have a div-element to attach it's drag handle or something like that. Without any comment I can see myself removing the extra div as cleanup a couple of weeks from now :)